contracts/ERC20Verifier.sol

```diff
@@ -26,24 +26,6 @@ contract ERC20Verifier is ERC20, ZKPVerifier {
         ERC20(name_, symbol_)
     {}
 
-    /**
```

Comment:
I have removed some logic here, in the tutorial, the point of the SC is to Airdrop tokens after verification,

this constract needs to be adjusted